app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import NotVaccinatedError, NotWearingMaskError
+from app.errors import OutdatedVaccineError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not ("vaccine" in visitor):
+            raise NotVaccinatedError(
+                "The visitor is not vaccinated"
+            )
+
+        vaccine_date = visitor["vaccine"]

```

Comment:
Can you explain why vaccine_date name?
As for me inside visitor["vaccine"] presents info about vaccine not just vaccive_date
vaccine_date is mean that vaccine_date is equal already to date